src/schema/sources.ts

```diff
@@ -1232,8 +1232,8 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       graphorm.queryOneOrFail(ctx, info, (builder) => ({
         ...builder,
         queryBuilder: builder.queryBuilder.where(
-          `"${builder.alias}"."id" = :id`,
-          { id },
+          `(CAST("${builder.alias}".id AS text) = :id OR ${builder.alias}.slug = :id)`,
+          { id: id.toLowerCase() },
         ),
```

Comment:
I think you can also keep the id as is, but add `orWhere()` to not have to do cast.

Reply:
Also would say no need to lower case, we expect lowercase from frontend, everything else is invalid slug.